Source/WebCore/layout/formattingContexts/grid/GridLayout.cpp

```diff
@@ -142,6 +142,33 @@ auto GridLayout::placeGridItems(UnplacedGridItems& unplacedGridItems, const Vect
     return Result { implicitGrid.gridAreas(), implicitGrid.columnsCount(), implicitGrid.rowsCount() };
 }
 
+static GridItemRects computeGridItemRects(const PlacedGridItems& placedGridItems, const BorderBoxPositions& inlineAxisPositions,
```

Comment:
The amount of parameters is a lot, but I am not sure we can really avoid that.

Reply:
There's probably a couple of ways we can clean it up but I didn't really want to think about it right now :P